docs/source/deployment/docker.rst

```diff
@@ -213,8 +213,11 @@ Isaac Lab Image Extensions
 The produced image depends on the arguments passed to ``container.py start`` and ``container.py stop``. These
 commands accept an image extension parameter as an additional argument. If no argument is passed, then this
 parameter defaults to ``base``. Currently, the only valid values are (``base``, ``ros2``).
-Only one image extension can be passed at a time. The produced container will be named ``isaac-lab-${profile}``,
-where ``${profile}`` is the image extension name.
+Only one image extension can be passed at a time. The produced image and container will be named
```

Comment:
Might make sense to move this to a separate paragraph to not overclutter the information here.